docs/testing.md

```diff
@@ -121,40 +121,21 @@ Sauce Labs can be used to run tests remotely in the cloud.
 
 1. Log into [http://saucelabs.com/account](http://saucelabs.com/account).
 
-2. [Download Sauce Connect](https://wiki.saucelabs.com/display/DOCS/Basic+Sauce+Connect+Proxy+Setup#BasicSauceConnectProxySetup-SettingUpSauceConnect)
-
-3. Open a new Terminal window or tab and navigate to the downloaded SauceConnect folder.
-    If you place the folder in your Application's folder this might look like:
-
-    ```
-    cd /Users/<YOUR MAC OSX USERNAME>/Applications/SauceConnect
-    ```
-
-4. Copy step 3 from the the SauceLabs
-   [Basic Setup instructions](https://wiki.saucelabs.com/display/DOCS/Basic+Sauce+Connect+Proxy+Setup#BasicSauceConnectProxySetup-SettingUpSauceConnect)
-   and run that in your Terminal window.
-   Once you see `Sauce Connect is up` in the Terminal,
-   that means the tunnel has successfully been established
-
-    > The Terminal command should already have your Sauce username and access key filled in.
-      If it doesn't, make sure you're logged in.
-
-5. Update and uncomment the `SAUCE_USERNAME`, `SAUCE_ACCESS_KEY`,
+2. Update and uncomment the `SAUCE_USERNAME`, `SAUCE_ACCESS_KEY`,
    and `SAUCE_SELENIUM_URL` values in your `.env` file.
    The access key can be found on the Sauce Labs
    [user settings page](https://saucelabs.com/beta/user-settings).
 
-6. Reload the settings with `cd .. && cd cfgov-refresh`. Type `y` if prompted.
+3. Reload the settings with `cd .. && cd cfgov-refresh`. Type `y` if prompted.
 
-7. Run the tests with `gulp test:acceptance --sauce`.
+4. Run the tests with `gulp test:acceptance --sauce`.
 
-8. Monitor progress of the tests
+5. Monitor progress of the tests
    on the [Sauce Labs dashboard](https://saucelabs.com/dashboard) Automated Tests tab.
 
 !!! Note
     If you get the error `Error: ENOTFOUND getaddrinfo ENOTFOUND`
     while running a test, it likely means that Sauce Connect is not running.
```

Comment:
I think this note could probably be removed as it was there when manually starting Sauce Connect, but now that that is automated it will be much harder to get into the circumstances where it's not running. It doesn't hurt to leave it either, up to you.

Reply:
I left this as-is for now